app/models/pet.rb

```diff
@@ -10,4 +13,12 @@ def shelter_name
   def self.adoptable
     where(adoptable: true)
   end
+
+  def self.search(name)
```

Comment:
This is correct implementation, but one of the benefits of working in a brownfield project is getting to take advantage of some of the existing methods. In the `application_record.rb` file (the parent class to every  model), there is a `search` method defined, which is very similar to this! So, you can use that search function instead and you don't need one implemented in the Pet model